code/modules/client/preferences.dm

```diff
@@ -547,6 +547,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 				else
 					HTML += "<font color=red>[rank]</font></td><td><font color=red><b> \[NON-HUMAN\]</b></font></td></tr>"
 				continue
+			if((rank == "AI") && !(config.allow_ai))
+				HTML += "<font color=red>[rank]</font></td><td><font color=red><b> \[DISABLED\]</b></font></td></tr>"
+				continue

```

Comment:
Wait a second, This is the occupation prefs screen. This is setting that you'd like to roll for a job. Any href exploit on this wouldnt actually do anything. If it did, then you'd be able to bypass jobbans just by loading your preferences, or bypass nonhuman by setting yourself to human, setting a job to high, and then going back to nonhuman.
This isn't actually going to fix the problem at all. Anyone who has AI set is still going to be able to roll for and get it. This is just visual on the menu.